CefSharp/IBrowserHost.cs

```diff
@@ -11,6 +11,14 @@ public interface IBrowserHost : IDisposable
     {
         void StartDownload(string url);
         void Print();
+        /// <summary>
+        /// Prints the current browser contents to the PDF file and executes a callback
+        /// on completion. The caller is responsible for deleting the file when done.
+        /// </summary>
+        /// <param name="path">PDF file location.</param>
+        /// <param name="settings">Settings.</param>
+        /// <param name="callback">The callback executed on completion.</param>
+        void PrintToPDF(string path, CefSharpPdfPrintSettings settings, IPrintToPdfCallback callback);
```

Comment:
What do you think about removing the callback and wrapping this in a Task? Can return a simple structure. Would be similar to GetTextAsync()

https://github.com/cefsharp/CefSharp/blob/bda5e9eeea52226aba3b0eb57438030e7f50d655/CefSharp.Core/Internals/CefFrameWrapper.cpp#L123